tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -3158,6 +3158,9 @@ Status ReplaceSoftplusTanhAndMulWithMish(
 bool RuntimeFusionEnabled(const Cluster& cluster) {
   static bool is_enabled = [&] {
 #if CUDNN_VERSION >= 8400
+    // Cudnn runtime fusion feature is recommended for Ampere GPUs or later.
+    // For pre-Ampere GPUs, the overhead of runtime compilation would be very
+    // large and there are more limitations of supported cases.
```

Comment:
Is there a citation or something?  This is a real bummer, I've been working with nvidia on this feature literally for years, and now that we finally get it...it's not going to apply to Waymo at all.  :( :(

Reply:
Sorry about this. Answered above.